### PR TITLE
WIP [LPTOCPCI-346] Deploy ROSA hypershift cluster.

### PR DESCRIPTION
**Pull request by ascerra**

### CODE REVIEW

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.14-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.14-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.13-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.13-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.13-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-redhat-developer-service-binding-operator-release-v1.3.x-4.13-aws-rosa-hypershift-lp-interop-sbo

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.13-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-redhat-developer-service-binding-operator-release-v1.3.x-4.13-aws-rosa-hypershift-lp-interop-sbo

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-hypershift-poc-4.13-aws-rosa-hypershift-lp-interop-poc

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-redhat-developer-service-binding-operator-release-v1.3.x-4.13-aws-rosa-hypershift-lp-interop-sbo

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x__4.13.yaml [hunk 1]
```diff
@@ -68,6 +101,46 @@ tests:
...
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"},
+          {"step": "sbo-", "failure_type": "all", "classification":  "Test Failure", "jira_project": "APPSVC"}
```
Comment (calebevans) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x__4.13.yaml:116]:
```suggestion
          {"step": "sbo-*", "failure_type": "all", "classification":  "Test Failure", "jira_project": "APPSVC"}
```

**Comment by ascerra** [comment]
This PR is being left up as a PoC and will not be getting merged.

**Pull request closed without merging**